Quran/Base.lproj/Android.strings

```diff
@@ -278,3 +278,4 @@ Please note that we receive lots of emails, so we may not be able to reply to al
 "bookmarks_list_empty" = "No Bookmarks";
 "tags_list_empty" = "No Tags";
 "audio_manager" = "Audio Manager";
+"write_review" = "Write App Store Review";
```

Comment:
Can you use `Localizable.strings` file instead of `Android.strings`, since `Android.strings` is the translations coming from the Android app.